index.html

```diff
@@ -46,13 +52,34 @@ <h1 class="logo__text">Renome</h1>
                                     <a href="#" class="header__link">menu</a>
                                 </li>
                                 <li>
-                                    <a href="#" class="header__link">reservation</a>
+                                    <a href="#" class="header__link">reservations</a>
                                 </li>
                                 <li>
                                     <a href="#" class="header__link">blog</a>
                                 </li>
                                 <li>
-                                    <a href="" class="header__link">features</a>
+                                    <a class="header__link" id="submenu">features</a>
+                                        <button class="header__linkbutton" id="button_submenu">
+                                            <i class="arrow_right"></i>
+                                        </button>
+                                    <ul class="header__subnav" id="sub_nav">
+                                        <li>
+                                            <a>
+                                                <button class="header__sublink" id="button_back">
+                                                    <i class="arrow_left"></i>
```

Comment:
why `<i>` ?

Reply:
I needed any element to make an arrows > < , and I choose <i> because nowhere else in this Renome-site we don't use italic

Reply:
Changed "i" to "div".